src/createStore.js

```diff
@@ -115,6 +116,8 @@ export default function createStore(reducer, initialState, enhancer) {
 
       isSubscribed = false
 
+      listener()
```

Comment:
Wait, why are you calling the listener on unsubscribe? That's unexpected behavior.